src/engine-actions/get-total-balance-for-address.js

```diff
@@ -0,0 +1,24 @@
+const { Big } = require('../utils')
+const getChannelsForRemoteAddress = require('./get-channels-for-remote-address')
+
+/**
+ * Get local balance of all channels for a specific address
+ * @param {string} address
+ * @returns {string} totalBalance (int64)
+ */
+async function getTotalBalanceForAddress (address) {
+  const channelsForAddress = await getChannelsForRemoteAddress.call(this, address)
+
+  if (channelsForAddress.length === 0) {
+    this.logger.debug('getTotalBalanceForAddress: No open or pending channels exist', { symbol: this.symbol, address })
+    return Big(0).toString()
```

Comment:
[nit] Why use `Big(0).toString()` instead of just '0'?

Reply:
More consistent with other values, but it is really no different.